android/app/src/main/java/com/wbooks/MainApplication.java

```diff
@@ -4,6 +4,10 @@
 import android.content.Context;
 import com.facebook.react.PackageList;
 import com.facebook.react.ReactApplication;
+import com.th3rdwave.safeareacontext.SafeAreaContextPackage;
+import com.swmansion.rnscreens.RNScreensPackage;
+import com.swmansion.reanimated.ReanimatedPackage;
+import com.swmansion.gesturehandler.react.RNGestureHandlerPackage;
```

Comment:
If you added this I think is not necessary because of the `react native auto link`